pennylane/operation.py

```diff
@@ -1025,6 +1029,29 @@ def generator(self):  # pylint: disable=no-self-use
         """
         raise GeneratorUndefinedError(f"Operation {self.name} does not have a generator")
 
+    def pow(self, n):
+        """A new operator equal to this one raised to the given power.
+
+        This *developer* method should be used to define shortcuts and easier decompositions.
+        For example, this method may specify how a Pauli gate raised to an even power is equal
+        to nothing, or we can exponentiate a rotation gate by multiplying the angle by the exponent.
```

Comment:
To nothing? :) The identity?

Just a general thought: I used to prefer docstrings that explain applications, but I changed my mind a while back. Docstrings should just explain what the method does, in 3 year's time our ideas for use cases may be totally irrelevant, but the explanation will never expire.

Reply:
In other words: I'd delete it